jib-cli/src/main/java/com/google/cloud/tools/jib/cli/jar/JarModeProcessor.java

```diff
@@ -151,6 +116,32 @@ static JarType determineJarType(Path jarPath) throws IOException {
     return layers;
   }
 
+  /**
+   * Creates layers for dependencies, snapshot dependencies and the jar itself on container for a
+   * standard jar.
+   *
+   * @param jarPath path to jar file
+   * @return list of {@link FileEntriesLayer}
+   * @throws IOException if I/O error occurs when opening the jar file
+   */
+  static List<FileEntriesLayer> createPackagedModeLayersForStandardJar(Path jarPath)
+      throws IOException {
+    // Add dependencies layers.
+    List<FileEntriesLayer> layers = getDependenciesLayers(jarPath);
+
+    // Add layer for jar.
+    FileEntriesLayer jarLayer =
+        FileEntriesLayer.builder()
+            .setName(JAR)
+            .addEntry(
+                jarPath,
+                APP_ROOT.resolve(RelativeUnixPath.get("jar")).resolve(jarPath.getFileName()))
```

Comment:
Oh, I remember we decided to do `java -jar`. In that case, we should retain the directory structure for the dependencies given in `Class-Path:` in `MANIFEST.MF`.

Reply:
Actually, I do see the dependency directory structure is retained under `/app/dependencies`. But for the `java -jar` to work, the easiest solution is to put the main JAR under `/app/dependencies`. But then, the directory name `/app/dependencies` doesn't make much sense. Maybe we should place the main JAR and other dependency JARs (while retaining the directory structure) at `/app`?

That said, it's worth having a test that has `Class-Path:`, e.g., `Class-Path: foo.jar nested/bar.jar`.